class02/exercise_6.py

```diff
@@ -82,7 +93,30 @@ def perfect_number():
 	'''
 
 	# code up your answer here
-
-
+	def is_prime_number(x):
+    		if x >= 2:
+        		for y in range(2,x):
+           			 if not ( x % y ):
+                			return False
+   	 	else:
+        		return False
+    		return True;
+	
+	def multiplyList(myList) : 
```

Comment:
seems to be doing what `numpy.prod` is supposed to do. check it out: https://docs.scipy.org/doc/numpy/reference/generated/numpy.prod.html

Reply:
This is very helpful. Thank you!